config.json

```diff
@@ -1242,7 +1249,7 @@
         "name": "Binary Search",
         "uuid": "7c569e5d-bb00-44b8-8adc-34253790c19b",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["arrays", "conditionals", "math", "while-loops"],
```

Comment:
`errors` is missing and I would add `comparison` as well since there are lots of variants of comparison operators in this exercise